Plugin/Model/MenuBuilderCommand.php

```diff
@@ -37,7 +37,7 @@ public function afterExecute(AbstractCommand $subject, $result): array
         if ((isset($result['id']) && $this->isAllowedMenuId->isAllowed($result['id'])) ||
             (
                 isset($result['module']) &&
-                !isset($result['parent']) &&
+                !($result['parent'] ?? '') &&
```

Comment:
this will check that the value of parent is null or an empty string

Reply:
Thanks for the fix